sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/TestMethodShouldContainAssertionTest.cs

```diff
@@ -142,8 +155,51 @@ public void TestMethodShouldContainAssertion_Xunit_Legacy()
             Verifier.VerifyAnalyzer(@"TestCases\TestMethodShouldContainAssertion.Xunit.Legacy.cs",
                 new TestMethodShouldContainAssertion(),
                 additionalReferences: NuGetMetadataReference.XunitFrameworkV1
-                    .Concat(NuGetMetadataReference.FluentAssertions("1.6.0"))
+                    .Concat(NuGetMetadataReference.FluentAssertions(Constants.NuGetLatestVersion))
+                    .Concat(FrameworkMetadataReference.SystemXml)
+                    .Concat(FrameworkMetadataReference.SystemXmlLinq)
                     .ToArray());
         }
+
+        [DataTestMethod]
+        [DataRow(NUnitVersions.Ver25, FluentAssertionVersions.Ver1)]
```

Comment:
curiosity (no action needed) : was there a reason not to combine other `NUnitVersions` in this test?

Reply:
The main reason is that the nunit version should not be relevant for this test because what we want to verify is that we support old versions of FluentAssertions.

A second reason is that by using the latest version of NUnit I get an error message: "Issue with message 'CS0012: The type 'Attribute' is defined in an assembly that is not referenced. You must add a reference to assembly 'netstandard...."